src/helpers/elm/ELMInfoCache.ts

```diff
@@ -35,7 +34,7 @@ const ELMInfoCache = new Map<string, ELMInfoCacheType>();
  * @returns {Object} an object containing cqls, elmJSONs, rootLibIdentifier, codeService, rep, and vsMap
  */
 export function retrieveELMInfo(
-  measure: fhir4.Measure,
+  measure: MeasureBundleHelpers.MeasureWithLibrary,
```

Comment:
```suggestion
  measure: MeasureWithLibrary,
```
And then import it from `MeasureBundleHelpers.ts`? This would be consistent with what's in `Execution.ts`, but it doesn't really matter.